webhook/configmaps/configmaps.go

```diff
@@ -184,6 +184,15 @@ func (ac *reconciler) validate(ctx context.Context, req *admissionv1beta1.Admiss
 		}
 	}
 
+	exampleData, hasExampleData := newObj.Data[configmap.ExampleKey]
+	exampleHash, hasExampleHashLabel := newObj.Labels[configmap.ExampleHashLabel]
+	if hasExampleData && hasExampleHashLabel &&
+		exampleHash != configmap.ExampleHash(exampleData) {
+		return fmt.Errorf(
+			"%q block edited, you likely wanted to create an unindented configuration",
+			configmap.ExampleKey)
```

Comment:
I wonder if we should include the diff 🤔

Reply:
Diff of the update or diff of the hash?